capstone-frontend/src/App.jsx

```diff
@@ -22,27 +21,45 @@ class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      searchQuery: ""
+      searchQuery: "",
+      bookLists: []
     };
   }
 
   setSearchQuery = (value) => {
     if (value !== this.state.searchQuery) {
-      this.setState({ searchQuery:value });
+      this.setState({ searchQuery: value });
     }
   }
 
+  fetchBookLists = async () => {
+
+    const userID = window.localStorage.getItem("userID");
+
+    const bookLists = await fetch(`/api/booklist?userID=${userID}`, {
+      method: "GET",
+    }).then(resp => resp.json());
```

Comment:
It may be good to have a .catch() here so that we can manage errors should they come up from the GET request